### PR TITLE
Add isExactMatchRequired property

### PR DESCRIPTION
**Pull request by emzho**
API review ID: 14849
Roadmap item ID: 15593

### CODE REVIEW

File: changelog/Microsoft.Search.json [hunk 1]
```diff
@@ -1405,6 +1405,24 @@
...
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "ce913b61-7449-4839-ab8d-fa3c74ba9f1a",
+          "ApiChange": "Property",
+          "ChangedApiName": "isExactMatchRequired",
+          "ChangeType": "Addition",
+          "Description": "Added the **isExactMatchRequired** property to the [property](https://learn.microsoft.com/en-us/graph/api/resources/property?view=graph-rest-beta) entity.",
```
Comment (isvargasmsft) [inline changelog/Microsoft.Search.json]:
```suggestion
          "Description": "Added the **isExactMatchRequired** property to the [property](https://learn.microsoft.com/en-us/graph/api/resources/property?view=graph-rest-beta) resource.",
```

File: concepts/connecting-external-content-manage-schema.md [hunk 2]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
 
 *Refine results by `tags`, a refinable property.*
 
+### Exact Match Required
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:76]:
```suggestion
### Exact match required
```

File: concepts/connecting-external-content-manage-schema.md [hunk 3]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+
+If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can specify exact matching.
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:78]: Line 78, second sentence: It sound like the property is the one that specifies. Could you please rephrase to indicate what is the desired action from the developer? How can the developer specify this? 

Sugestion:
```suggestion
If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can by specified for exact matching.
```

File: concepts/connecting-external-content-manage-schema.md [hunk 4]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+
+For example, the `ticketId` property is both queryable and specifies exact matching.
+- Querying "ticketId:CTS-ce913b61" will return the item with a ticket ID property "CTS-ce913b61"
+- Querying "ticketId:CTS" will NOT return the item with ticket ID "CTS-ce913b61"
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:82]:
```suggestion
- Querying "ticketId:CTS-ce913b61" will return the item with a ticket ID property "CTS-ce913b61".
- Querying "ticketId:CTS" will NOT return the item with ticket ID "CTS-ce913b61".
```

File: concepts/connecting-external-content-manage-schema.md [hunk 5]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+
+Similarly, the `tags` property also specifies exact matching.
+- Querying "tags:contoso" will return any item with the tag "contoso"
+- Querying "tags:contoso" will NOT return items with the tag "contoso ticket"
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:86]:
```suggestion
- Querying "tags:contoso" will return any item with the tag "contoso".
- Querying "tags:contoso" will NOT return items with the tag "contoso ticket".
```

File: concepts/connecting-external-content-manage-schema.md [hunk 6]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+
+For example, there may be a scenario where the item property is a GUID-formatted string. If this property must be matched exactly for item queries, then please specify that `isExactMatchRequired` is true.
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:88]: In general, we avoid using the word "please" unless we are requesting the user to perform an inconvenient action.
Details here: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please

```suggestion
For example, there may be a scenario where the item property is a GUID-formatted string. If this property must be matched exactly for item queries, specify that `isExactMatchRequired` is true.
```

File: concepts/connecting-external-content-manage-schema.md [hunk 7]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+
+The `title` property does not specify exact matching. If nothing is specified, then the `isExactMatchRequired` is false. The `title` property will be tokenized based on the item content language's tokenization rules.
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:90]: Suggestion:
```suggestion
The `title` property does not specify exact matching. If nothing is specified, then `isExactMatchRequired` is false. The `title` property will be tokenized based on the tokenization rules of the language of the item content.
```

File: concepts/connecting-external-content-manage-schema.md [hunk 8]
```diff
@@ -73,6 +73,22 @@ If a property is refinable, an admin can configure it as a custom filter in the
...
+- Querying "title:Contoso Title" will return any item containing "Contoso" or "Title" in the `title` property
```
Comment (isvargasmsft) [inline concepts/connecting-external-content-manage-schema.md:91]:
```suggestion
- Querying "title:Contoso Title" will return any item containing "Contoso" or "Title" in the `title` property.
```

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @emzho, 

Very good job with this PR, very complete. 

Please take a look at a couple of suggestions we are adding to the conceptual topic. Feel free to ping me on the side if you have any questions. 

Kind regards, 

Isaac Vargas

**Review by isvargasmsft — APPROVED** [review]

File: concepts/connecting-external-content-manage-schema.md (hunk 3 above)
Comment (dodaromike) [reply concepts/connecting-external-content-manage-schema.md]: There is a typo in this revision: 
If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can >>by<< specified for exact matching.
Should be: If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can be specified for exact matching.

Comment (isvargasmsft) [reply concepts/connecting-external-content-manage-schema.md]: Great catch @dodaromike. Fixed. Thank you!

**Pull request merged**